src/ffi/mod.rs

```diff
@@ -567,10 +587,6 @@ pub extern "C" fn telio_connect_to_exit_node(
     allowed_ips: *const c_char,
     endpoint: *const c_char,
 ) -> telio_result {
-    telio_log_info!(
```

Comment:
Is there a reason to move it?

Reply:
`telio_connect_to_exit_node()` is only a wrapper around `telio_connect_to_exit_node_with_id()`. Previously only `telio_connect_to_exit_node()` was logged, and we were unable to see any calls to `telio_connect_to_exit_node_with_id()`. That method is being used on android, meaning, that we were unable to see the fact that android app is initiating VPN connection. First debugging attempt lead us to the fact, that DNS is enabled, but VPN is not. Later it turned out that only log line is missing.